src/p2p/Join/routes.ts

```diff
@@ -484,6 +540,8 @@ const gossipUnjoinRequests: P2P.P2PTypes.GossipHandler<UnjoinRequest, P2P.NodeLi
   sender: P2P.NodeListTypes.Node['id'],
   tracker: string
 ) => {
+  // unjoin requests should be received at all quarters
```

Comment:
this is not longer going to be true after RED-13 is merged. go ahead and the same quarter checks to this handler as well. we will just have to ensure that this is merged after RED-13